automation-labs/aws/aws-s3-bucket-automation.md

```diff
@@ -42,6 +42,7 @@ In this exercise you will install and configure the AWS CLI.
    - *Default region name [None]:* `us-west-2`
    - *Default output format [None]:* `JSON`
 
+If you do not have your access key and secret access key, you will need to contact your instructor since this information is only generated once, and not available for future viewing.
```

Comment:
The instructor will not be able to help. So this sentence is not helpful. Could you please remove.

Reply:
Removed this change.